.env.sample

```diff
@@ -6,11 +6,12 @@ EMAIL_PASS=
 EMAIL_SERVICE=
 EMAIL_USER=
 GDRIVE_CLIENT_EMAIL=
-GDRIVE_CLIENT_ID=
+GDRIVE_PROJECT_ID=
```

Comment:
I don't think there should be 2 GDRIVE_PROJECT_ID s

Reply:
I think this used to be GDRIVE_CLIENT_ID but GDRIVE_CLIENT_ID is not used anywhere in the code. I would just remove this line.
```suggestion
```